internal/fsx/fsx_test.go

```diff
@@ -0,0 +1,74 @@
+package fsx_test
+
+import (
+	"bufio"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/ooni/probe-engine/internal/fsx"
+)
+
+var StateBaseDir = "../../testdata/fsx/"
```

Comment:
```suggestion
var StateBaseDir = "./testdata/fsx/"
```

I suggest to create this directory as `internal/fsx/testdata`, so we have less cross references between different parts of the codebase, which is in general a very nice property to have.